app/controllers/reports_controller.rb

```diff
@@ -8,7 +8,7 @@ def donations_summary
 
   def manufacturer_donations_summary
     @recent_donations_from_manufacturers = current_organization.donations.during(helpers.selected_range).by_source(:manufacturer)
-    @top_manufacturers = current_organization.manufacturers.by_donation_count(10, helpers.selected_range)
+    @top_manufacturers = current_organization.manufacturers.by_donation_date(10, helpers.selected_range)
```

Comment:
I'm not a real PR reviewer, so take any of my comments with a grain of salt.

But I think the `@top_manufacturers` instance variable could be renamed here to something like `@recent_manufacturers` (or similar) because we are now sorting them by their last donation's issue date. And it would match the change you mean in the view text for this report.